### PR TITLE
add sysoid for easy_ups

### PR DESCRIPTION
**Pull request by pkanakaraj19**

### CODE REVIEW

File: profiles/kentik_snmp/apc/apc_ups.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ extends:
 provider: kentik-ups
 
 sysobjectid:
+  - 1.3.6.1.4.1.935.1.1.1     # APC EasyUPS
```
Comment (thezackm) [inline profiles/kentik_snmp/apc/apc_ups.yml:10]: @pkanakaraj19 - this OID vendor code `1.3.6.1.4.1.935` is [owned by a different company than APC](https://oidref.com/1.3.6.1.4.1.935). Are you positive that this is an APC device?

Comment (pkanakaraj19) [reply profiles/kentik_snmp/apc/apc_ups.yml:10]: @thezackm that's the OID I'm seeing in SNMP walk output, and in the New Relic entity summary page (currently classified as 'kentik_default'). This is the NMC used on the device: https://www.apc.com/us/en/product/E3SOPT001/network-card-for-easy-ups-3s-3m-3l-and-galaxy-pw-2nd-gen-ups-/

Comment (pkanakaraj19) [reply profiles/kentik_snmp/apc/apc_ups.yml:10]: @thezackm confirmed with a person on site where the UPS is located - it's an APC Easy UPS 3M model with the above linked NMC.

Comment (pkanakaraj19) [reply profiles/kentik_snmp/apc/apc_ups.yml:10]: I'll connect with APC support and see if the NMC needs an update. Please hold off on the PR and thanks for pointing it out! @thezackm

Comment (thezackm) [reply profiles/kentik_snmp/apc/apc_ups.yml:10]: gotcha; note that a lot of times this happens when a vendor buys another company; it's definitely plausible. will mark this to hold until you get back; thanks.

**Pull request closed without merging**